src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -1981,4 +2008,28 @@ public int getLastContactInHours()
     {
         return lastContactInHours;
     }
+
+    @Override
+    public boolean hasWillRaidTonight()
+    {
+        return willRaidTonight;
+    }
+
+    public void setWillRaidTonight(final Boolean willRaid)
+    {
+        willRaidTonight = willRaid;
+    }
+
+    @Override
+    public List<EntityLivingBase> getGuardTargets()
+    {
+        final List<EntityLivingBase> targets = guardTargets;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "targets". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1488)